src/main/daml/Daml/Finance/Interface/Types/Common.daml

```diff
@@ -21,6 +22,15 @@ type Parties = Set Party
 -- Allows for processes to add/remove parties for their specific purpose.
 type PartiesMap = Map Text Parties
 
+-- | Parties involved in transfers.
+data Transferors = Transferors
+  with
+    instructors : Parties
+      -- ^ Parties instructing a transfer (outgoing).
+    approvers : Parties
+      -- ^ Parties approving a transfer (incoming).
+  deriving (Eq, Ord, Show)
+
```

Comment:
Curious as to why this needs to be in the `Types` module?

Reply:
agree, will move it to `Account.I`